src/components/SearchBox/SearchBox.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import './SearchBox.scss';
+
+class SearchBox extends Component {
+  constructor(props) {
+    super();
+
+    this.props = props;
+    this.state = {
+      value: '',
+    };
+
+    this.onInputChange = this.onInputChange.bind(this);
+  }
+
+  onInputChange(event) {
```

Comment:
Keep function definitions consistent, I think it makes sense to keep things consistent and use the es6 function definition format here. Will also save us 3-4 lines of code.

`onInputChange = ({ target: { value } }) => this.setState({ value })`

Reply:
Sure - in that case the above comment is not needed.